docs/manual/mod/mod_ssl.xml

```diff
@@ -1816,6 +1816,97 @@ SSLStrictSNIVHostCheck on
 </usage>
 </directivesynopsis>
 
+<directivesynopsis>
+<name>SSLVHostSNIPolicy</name>
+<description>Set compatibility policy for SNI client access to virtual hosts.</description>
+<syntax>SSLVHostSNIPolicy strict|secure|authonly|insecure</syntax>
+<default>SSLVHostSNIPolicy secure</default>
+<contextlist><context>server config</context></contextlist>
+<compatibility>Available in httpd 2.5 and later</compatibility>
+
+<usage><p>This directive sets policy applied when checking whether the
```

Comment:
Question by a non native speaker: Doesn't this need to be:
```suggestion
<usage><p>This directive sets the policy applied when checking whether the
```

Reply:
Yes I think that's better, thanks!